port/common/omrstr.c

```diff
@@ -3169,6 +3169,7 @@ convertWideToPlatform(struct OMRPortLibrary *portLibrary, charconvState_t encodi
 			uintptr_t oldOsbSize = 0;
 			uintptr_t iconvResult = 0;
 			osbSize = sizeof(onStackBuffer);
```

Comment:
Also, `osbSize` (= `outbytesleft`) should not be reset within the `while` loop. The decrements to `osbSize` (= `outbytesleft`) performed by the `iconv()` function will be lost.

Reply:
Similar reason as above. This code hasn't been changed for years anyway.